lms/djangoapps/mobile_api/course_info/urls.py

```diff
@@ -19,4 +19,5 @@
         CourseUpdatesList.as_view(),
         name='course-updates-list'
     ),
+    url(r'^record_user_activity$', CourseGoalsRecordUserActivity.as_view(), name='record_user_activity'),
```

Comment:
Oh I hadn't noticed this before - you're taking all arguments as json rather than via the URL like `/record_user_activity/course-v1:edx+DemoX+Run/12345` - I think I like your version better, because there isn't an obvious ordering there.

I also dislike that we tend to use underscores rather than hyphens for our endpoint names. A Python-ism that crept into our REST design. But not something you need to change. I'm just talking out loud.